### ISSUE TITLE
Playback Status

### ISSUE THREAD

**Issue by bsakai2000** [bsakai2000/MusicController#1]
Add support for playback status. Maybe use `poll` in the app to listen for events from the addon? Or a second thread that just listens using `scanf` and updates a global to match every time it gets input?

**Comment by bsakai2000** [comment]
Sample PropertiesChanged signal

```
signal time=1596012056.363301 sender=:1.288 -> destination=(null destination) serial=3 path=/org/mpris/MediaPlayer2; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.mpris.MediaPlayer2.Player"
   array [
      dict entry(
         string "Volume"
         variant             double 0
      )
   ]
   array [
   ]
signal time=1596012060.293554 sender=:1.288 -> destination=(null destination) serial=4 path=/org/mpris/MediaPlayer2; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.mpris.MediaPlayer2.Player"
   array [
      dict entry(
         string "PlaybackStatus"
         variant             string "Stopped"
      )
   ]
   array [
   ]
```

**Comment by bsakai2000** [comment]
Fixed by 591ca854ae437c94653b929b941a377a94c509e3
